### PR TITLE
Add annotation to all cluster class templates

### LINKED ISSUE #3077: Add annotation to all cluster class templates
**Describe the feature request**

UI needs an annotation to recognize the cluster class and infra provider, to help the end user to fill details while creating cluster

**Describe alternative(s) you've considered**

**Affected product area (please put an X in all that apply)**

- ( ) APIs
- ( ) Addons
- (x) CLI
- ( ) Docs
- ( ) IAM
- ( ) Installation
- ( ) Plugin
- ( ) Security
- ( ) Test and Release
- ( ) User Experience
- ( ) Developer Experience

**Additional context**

### PR DESCRIPTION
**Pull request by chandrareddyp**
### What this PR does / why we need it
This PR adds an annotation to cluster class templates for aws, Azure and vSphere providers.

why we need it:
Annotation helps UI to recognize the cluster class and its variables to help end-user while creating cluster class based cluster
### Which issue(s) this PR fixes

Fixes #3077 

### Release note

```release-note
Version annotation has been added in ClusterClass templates
```

### CODE REVIEW

File: pkg/v1/providers/infrastructure-azure/v1.4.0/cconly/base.yaml [hunk 1]
```diff
@@ -7,6 +7,8 @@ kind: ClusterClass
 metadata:
   name: tkg-azure-default
   namespace: #@ data.values.NAMESPACE
+  annotations:
+    identifier.cc.vmware.com: tanzu-aws-default-0.0.1
```
Comment (tenczar) [inline pkg/v1/providers/infrastructure-azure/v1.4.0/cconly/base.yaml:11]:
```suggestion
    identifier.cc.vmware.com: tanzu-azure-default-0.0.1
```

File: packages/tkg-clusterclass-azure/bundle/config/upstream/base.yaml [hunk 2]
```diff
@@ -7,6 +7,8 @@ kind: ClusterClass
 metadata:
   name: tkg-azure-default
   namespace: #@ data.values.NAMESPACE
+  annotations:
+    identifier.cc.vmware.com: tanzu-aws-default-0.0.1
```
Comment (tenczar) [inline packages/tkg-clusterclass-azure/bundle/config/upstream/base.yaml:11]:
```suggestion
    identifier.cc.vmware.com: tanzu-azure-default-0.0.1
```

**Review by tenczar — CHANGES_REQUESTED** [review]
Small fix, then looks good.

File: packages/tkg-clusterclass-azure/bundle/config/upstream/base.yaml (hunk 2 above)
Comment (chandrareddyp) [reply packages/tkg-clusterclass-azure/bundle/config/upstream/base.yaml]: done. thanks.

File: pkg/v1/providers/infrastructure-azure/v1.4.0/cconly/base.yaml (hunk 1 above)
Comment (chandrareddyp) [reply pkg/v1/providers/infrastructure-azure/v1.4.0/cconly/base.yaml]: done. thanks.

**Review by tenczar — APPROVED** [review]
LGTM

**Review by anujc25 — APPROVED** [review]
LGTM. Please file corresponding issue and update the PR description.

**Comment by chandrareddyp** [comment]
> LGTM. Please file corresponding issue and update the PR description.

https://github.com/vmware-tanzu/tanzu-framework/issues/3077 created. thanks.

**Pull request merged**